src/liger_kernel/chunked_loss/fused_linear_preference.py

```diff
@@ -18,6 +18,30 @@ def preference_loss_fn(chosen_logps, rejected_logps, beta=0.1):
         """
         raise NotImplementedError("Preference loss function must be implemented.")
 
+    @staticmethod
+    def get_ref_logps(
+        input_chunk, ref_weight, target_chunk, ref_bias=None, ignore_index=-100
+    ):
+        with torch.no_grad():
+            ref_logits_chunk = input_chunk @ ref_weight.t()
```

Comment:
The logic to get the log_probs is common for both the reference and active policy model, wondering if we can create a util function that can be called by both instead of repeating the code.